templates/web/pages/product_edit/product_edit_form_process.tt.html

```diff
@@ -1,15 +1,44 @@
 <!-- start templates/[% template.name %] -->
-
+[%# This template is called after editing a product %]
 [%  IF display_random_sample_of_products_after_edits_options.defined && display_random_sample_of_products_after_edits_options %]
-
+    [%# Nothing todo we already display a product %]
 [% ELSIF type == 'delete' %]
 
     <p>[% lang("product_deleted") %]</p>
 
 [% ELSE %]
 
-    <p>[% lang("product_changes_saved") %]</p>
-    <p>&rarr; <a href="[% edited_product_url %]">[% lang("see_product_page") %]</a></p>
+    <section class="row">
```

Comment:
```suggestion
    <section class="row" id="changes_saved">
```
I think it's always useful to add ids to the main blocks, such as sections: for debugging, for third party scripts, etc.